ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/dialogs/RepositoryResult.java

```diff
@@ -0,0 +1,4 @@
+package org.eclipse.pde.internal.ui.dialogs;
```

Comment:
Please move this into the dialog class having this record standalone is Not necessary.